src/content/en/2022/capabilities.md

```diff
@@ -16,7 +16,372 @@ featured_stat_2: TODO
+
+{{ figure_markup(
+  image="Gamepad-API.webp",
+  gif="Gamepad-API.gif",
+  caption="Connecting a Xbox controller to Google Stadia in the Chrome browser.",
+  description="Connecting a Xbox controller to Google Stadia in the Chrome browser.",
+  width=640,
+  height=360
+) }}
+
+## Push API
+
+The [_Push API_](https://developer.mozilla.org/en-US/docs/Web/API/Push_API) allows web applications to receive messages from a server regardless of whether the application was in the foreground. Developers can send asynchronous notifications and updates to users who opt in, giving them meaningful updates and a nudge to reengage with an application.
+
+Web applications must also have a service worker to receive push notifications from a server. From within the service worker, push notifications can be subscribed to using the [`PushManager.subscribe()`](https://developer.mozilla.org/en-US/docs/Web/API/PushManager/subscribe).
+
+The Push API is available on modern versions of Chrome, Edge, and Firefox.
+
+### Usage of Push API
+
+{{ figure_markup(
+  caption="Usage of Push API on mobile.",
+  content="1.86%",
+  classes="big-number",
+  sheets_gid="1887140434",
+  sql_file="fugu.sql"
+)
+}}
+
+The Push API was not measured last year. In its first year of tracking, the API was used on 2.03% of desktop pages and 1.86% of mobile pages, making it the eighth most used capability on desktop and mobile.
+
+## Project Fugu
+
+Many features users expect to belong to platform-specific applications also exist on the web. However, thanks to the Capabilities Project, known by many as Project Fugu, these features exist on the web. Project Fugu is a cross-company effort to bring feature parity to web applications, considering what iOS, Android, or desktop apps can do. Project Fugu works on exposing platform-specific capabilities to the web while maintaining user security, privacy, trust, and the web's other core tenets.
+
+Project Fugu comprises Microsoft, Intel, Samsung, Google, and many other groups and individuals.
+
+Check out <a hreflang="en" href="https://developer.chrome.com/blog/fugu-status/">this post</a> on the Chrome Developers blog to learn more about the Capabilities Project.
+
+## Conclusion
+
+Capabilities unlock new possibilities and functionality for developers to take advantage of on the web. This chapter shared eight of the most popular web platform APIs currently being used on the web. This chapter also showcased some of these capabilities used in different web applications. The beauty of the web is that it can use these platform-based functionalities without needing to be installed onto a device or additional libraries and plugins.
```

Comment:
```suggestion
Capabilities unlock new possibilities and functionality for developers to take advantage of on the web. This chapter shared eight of the most popular web platform APIs currently being used on the web. It also showcased some of these capabilities used in different web applications. The beauty of the web is that it can use these platform-based functionalities without needing to (necessarily) be installed onto a device or additional libraries and plugins.
```